models/template_model.go

```diff
@@ -0,0 +1,144 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+import (
+	"database/sql/driver"
+	"time"
+
+	"github.com/percona-platform/saas/pkg/common"
+	"gopkg.in/reform.v1"
+)
+
+//go:generate reform
+
+// Template represents Integrated Alerting rule template.
+//reform:ia_templates
+type Template struct {
+	Name        string        `reform:"name,pk"`
+	Version     uint32        `reform:"version"`
+	Summary     string        `reform:"summary"`
+	Tiers       Tiers         `reform:"tiers"`
+	Expr        string        `reform:"expr"`
+	Params      Params        `reform:"params"`
```

Comment:
shouldn't this be a slice? https://github.com/percona/pmm/blob/PMM-2.0/api/managementpb/ia/templates.pb.go#L446

Reply:
nevermind, I missed the type alias